antismash/detection/hmm_detection/cluster_rules/loose.txt

```diff
@@ -20,3 +20,9 @@ RULE fatty_acid
     CUTOFF 20
     NEIGHBOURHOOD 10
     CONDITIONS  bt1fas or ft1fas or t2fas or fabH
+
+RULE halogenated
+    COMMENT why
```

Comment:
Halogenases are frequently involved in secondary metabolite biosynthesis.

Reply:
Sounds good, I'll use a modified version of that for the docs description.